jina/executors/indexers/__init__.py

```diff
@@ -162,17 +162,31 @@ def flush(self):
         except:
             pass
 
-    def _filter_nonexistent_keys(self, keys: Iterator, existent_keys: Iterator, check_path: str):
-        indices_to_drop = []
+    def _filter_nonexistent_keys_values(self, keys: Iterator, values: Iterator, existent_keys: Iterator, check_path: str) -> Tuple[List, List]:
```

Comment:
can the typehints be more specific as per which kind of iterators it gets?

Reply:
Not really. Each of the indexers have their own types. I had to add IgnoreWarnings comment on some